README.md

```diff
@@ -63,7 +65,38 @@ The valid project types are:
 - `classlib` for Class Library
 - `unittest` Unit Test project
 
-> Example: `yo aspnet webbasic "my test app"` will create a "Web Application Basic" project called "my test app"
+The valid UI framework types are:
+
+- `bootstrap` for Bootstrap (this is the default and does not have to be specified explicitly)
+- `semantic` for Semantic UI
+
+> Example: `yo aspnet webbasic "my semantic app" semantic` will create a "Web Application Basic" project called "my semantic app" using the Semantic UI framework.
+
+> Example: `yo aspnet webbasic "my bootstrap app"` OR `yo aspnet webbasic "my bootstrap app" bootstrap` will create a "Web Application Basic" project called "my bootstrap app" using the Bootstrap framework.
+
+## Additional UI Framework Notes
+
```

Comment:
This is too long and too detailed IMHO. When people quickly scan readme's on NPM they want to got all information. The short note up to this point should be enough. As there is no release notes on `generator-aspnet` (yet), I think this long section should be moved/repeated in:
- dedicated wiki page describing Why's/Hows of Bootstrap/SemanticUI split. How they are differnet, Which comes by default and why, etc.
- a long PR/Commit desription - that is why PR/commits are extremelly usefull
- if merged I think you could write a blog post on MSDN describing whys/hows of these changes
tec.

There was already discussion/topic to review README.md and make it shorter and cleaner.